iothub/device/src/TransportSettings/AmqpConnectionPoolSettings.cs

```diff
@@ -22,6 +20,9 @@ public sealed class AmqpConnectionPoolSettings
         /// </remarks>
         private const uint DefaultPoolSize = 100;
 
+        // Iot hub allows up to 999 tokens per connection. Setting the threshold just below that.
+        internal const uint MaxDevicesPerConnection = 995;
```

Comment:
I am curious why not 998 here? Or maybe we don't care much about this?

Reply:
Dunno what the historical context was.